include/MobilityInterface/MobilityInterface.h

```diff
@@ -37,12 +37,12 @@ namespace libmobility{
   class Mobility{
   private:
     int numberParticles;
-    bool initialized = false;
     std::uint64_t lanczosSeed;
     real lanczosTolerance;
     std::shared_ptr<LanczosStochasticVelocities> lanczos;
     real temperature;
   protected:
+    bool initialized = false;
```

Comment:
Perhaps this is something each solver should create for themselves if they need it. This variable is intended to be used only by the Mobility class for its own book keeping.